src/simulate/main.py

```diff
@@ -60,28 +61,52 @@ def main():
             sys.exit(1)
 
     files = os.listdir(json_files_dir)
+    session = requests.session()
     logging.info('list of json files in the directory %s', files)
+    event_count = 0
     for i in range(run_count):
         for file in files:
             if file.endswith(".json"):
                 path_to_file = json_files_dir + "/" + file
                 # Deserialize Json data
                 json_event = read_json(path_to_file)
                 if json_event is None:
+                    logging.error('Problem in reading Json event')
```

Comment:
Log the path so there is some idea how to check out the problem.  It would be sufficient to log it in the read_json method.  It shouldn't be a problem to collect the information.  If there are lots of errors we need to fix it.  If there aren't then the logging won't be happening.

Reply:
called function 'read_json` is printing the path. I have removed the extra logging(on Line 74) from hear